temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -273,6 +292,83 @@ public Builder setSslContext(SslContext sslContext) {
       return this;
     }
 
+    /**
+     * Sets default SSL context parameters that can be used with the temporal cloud service. Users
```

Comment:
same question around "temporal cloud service" here

Reply:
This is something we should discuss. First I thought about calling this method `setDefaultSslContext` or something along those lines, but then I thought that this will likely be used only by our cloud service users (who else would use TLS with certificates heh...) and decided to call it `setCloudSslContext` and that's why comments reflect it as well.

Reply:
I think I've found a better name `setSslContextWith(...)` and `setInsecureSslContextWith(...)`.